hw2/ml_project/inference_service.py

```diff
@@ -0,0 +1,30 @@
+from prepare_model import BaselineModel
+
+from enum import Enum
+from fastapi import FastAPI
+from pydantic import BaseModel
+from typing import List
+
+import joblib
+import numpy as np
+import os
+
+
+app = FastAPI()
+model = BaselineModel.load(os.environ['MODEL_PATH'])
+
+
+class Vec(BaseModel):
+    vals: List[float]
```

Comment:
hint: in python 3.10 typing list and now not necessary add import List 

please, use list instead List in typing